src/content/advanced/security-policy-enforcement/index.md

```diff
@@ -20,20 +20,25 @@ last_review_date: 2023-03-21
 
 <!-- {{< platform_support_table aws="alpha=v17.2.0" aws="ga=v17.4.0">}} -->
 
-__Note__: this content is applicable only to clusters running Kyverno, with the PSS `baseline` and `restricted` policies deployed in `enforce` mode. This will become the default in future Giant Swarm clusters, but can already be enabled on demand. This page will be updated to indicate the platform version where this content becomes the standard.
+__Note__: this content is applicable to Giant Swarm v19.2.0 and above, which include PSS `baseline` and `restricted` policies deployed in `enforce` mode.
 
-To enforce security best practices, several Kyverno policies mapped to the [Kubernetes Pod Security Standards][k8s-pss] are pre-installed in Giant Swarm clusters.
+__Note__: additional information for cluster admins can be found in a separate cluster admin guide.
```

Comment:
What about adding the link?

Reply:
Yeah I originally added it, but markdownlint blows up because the page doesn't exist yet :sweat_smile: I'll add it in the second PR
https://github.com/giantswarm/docs/pull/1929#issuecomment-1750955093